commons/src/components/MessageBody.svelte

```diff
@@ -5,18 +5,73 @@
   does not affect the global component enclosing it -->
 <script>
   import * as DOMPurify from "dompurify";
+  import { getEventDispatcher } from "@commons/methods/component";
+  import { get_current_component, onMount } from "svelte/internal";
 
   export let message;
+
+  const dispatchEvent = getEventDispatcher(get_current_component());
+
+  async function downloadSelectedFile(event, file) {
+    event.stopImmediatePropagation();
```

Comment:
Do we need to have this here if we're already calling it at the callback assignment level? (e.g. `on:click|stopPropagation`)

Reply:
you're right! this is redundant. will update